Add data-streams toolset to Datadog MCP Server documentation (#36523)
DataDog/documentation: content/en/bits_ai/mcp_server/tools.md

- Help me debug why this widget is showing fractional values when it should be a count.
- How do I configure a timeseries to show both bars and lines?

## Data Streams

Tools for investigating Kafka workloads with [Data Streams Monitoring][27], including topic, broker, and client configuration inspection, Schema Registry browsing, and on-demand reads of Kafka messages through the Datadog Agent.

**Note**: All tools in this toolset require the [Kafka integration][28] to be configured for your Kafka clusters. If a call returns a setup error or an empty result, follow the [Kafka setup guide][28] to enable Data Streams Monitoring on your Agent before retrying.

### `list_kafka_topic_configs`
*Toolset: **data-streams***\
*Permissions Required: `APM Read`*\

> Lists Kafka topic configuration overrides for a cluster, such as `cleanup.policy`, `retention.ms`, and `max.message.bytes`. Optionally filtered to a single topic. Use this to check whether topic-level settings explain producer or consumer behavior.

Reviewer: ```suggestion
Lists Kafka topic configuration overrides for a cluster, such as `cleanup.policy`, `retention.ms`, and `max.message.bytes`. You can optionally filter to a single topic. Use this to check whether topic-level settings explain producer or consumer behavior.
```

- Show me the topic config for `checkout-orders`.
- What is the retention setting on the orders Kafka topic?
- Compare `max.message.bytes` across all topics in the payments cluster.

### `list_kafka_broker_configs`
*Toolset: **data-streams***\
*Permissions Required: `APM Read`*\
Lists Kafka broker-level configuration for a cluster, optionally filtered to a single broker. Use this to inspect broker tuning when consumers across multiple topics are affected.

- Show me the broker configs for my Kafka cluster.
- What's the `num.io.threads` setting on broker 3?
- Compare broker configs between two clusters.

### `get_kafka_client_configs`
*Toolset: **data-streams***\
*Permissions Required: `APM Read`*\
Returns Kafka client (producer and consumer) configurations for a cluster, including consumer-group settings and client-side tuning.

- Show me the consumer config for service `checkout-api` on the payments cluster.
- What's the `session.timeout.ms` for consumer group `orders-processor`?
- Compare producer configs across services on my Kafka cluster.

### `list_all_kafka_schemas`
*Toolset: **data-streams***\
*Permissions Required: `APM Read`*\

> Lists all Schema Registry subjects for a cluster's schema registry. Use this when you don't know the subject name to inspect.

Reviewer: ```suggestion
Lists all subjects for a cluster's schema registry. Use this when you don't know which subject name to inspect.
```

- List all Schema Registry subjects.
- What schemas are registered for the orders topic?

### `get_subject_kafka_schemas`
*Toolset: **data-streams***\
*Permissions Required: `APM Read`*\
Returns all schema versions for a single Schema Registry subject (schema body, ID, version, and references). Use this after `list_all_kafka_schemas` to inspect the version history of a specific subject.

- Show every version of the `orders-value` schema.
- What changed in the latest version of the `payments-key` schema?
- Get the schema body for `checkout-orders-value` version 7.

### `read_kafka_messages`
*Toolset: **data-streams***\
*Permissions Required: `APM Read` and `Data Streams Monitoring Capture Messages`*\

> Reads messages from a given Kafka cluster, topic, and partition through the Datadog Agent by dispatching a Remote Config action and polling for the agent's response. Rate-limited to 10 calls per minute per organization. Useful for inspecting message payloads when investigating poison messages or stuck consumer offsets.

Reviewer: ```suggestion
Reads messages from a given Kafka cluster, topic, and partition through the Datadog Agent by dispatching a Remote Config action and polling for the Agent's response. This tool is rate-limited to 10 calls per minute per organization, and is useful for inspecting message payloads when investigating poison messages or stuck consumer offsets.
```
